cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/templates/etl/api/realtime/RealtimeContext.java

```diff
@@ -20,12 +20,12 @@
 import co.cask.cdap.templates.etl.api.StageSpecification;
 
 /**
- * Context passed to the Source stages.
+ * Context passed to the Source and Sink stages of Realtime Adapter.
  */
-public interface SourceContext extends RuntimeContext {
+public interface RealtimeContext extends RuntimeContext {
```

Comment:
Since we removed StreamWriter, DatasetContext interfaces from SinkContext, both SourceContext and SinkContext became identical. So I replaced them with RealtimeContext.